R/check.R

```diff
@@ -24,7 +24,7 @@ convert_and_check_my_args <- function(options) {
   ## Conversions
   options <- within(options, {
     if (has("libpath")) libpath <- as.character(libpath)
-    if (has("repos")) stopifnot(is.character(repos))
+    if (has("repos")) repos <- repos
```

Comment:
Is this reversion intentional?

Reply:
Yeah, I decided that we just pass is as it is. The child process will error if it is not good. I guess I could also  just remove that line completely.